cobra/test/test_io/test_model_notes.py

```diff
@@ -0,0 +1,31 @@
+import pytest
```

Comment:
Those import have to be sorted alphabetically to have the CI pass.

Reply:
@cdiener, I have made the changes. 
Thanks.